examples/model_selection/plot_grid_search_refit_callable.py

```diff
@@ -8,7 +8,7 @@
 score while minimising the number of PCA components [1].
 
 The figure shows the trade-off between cross-validated score and the number
-of PCA components. The balanced case is when n_components=6 and accuracy=0.80,
+of PCA components. The balanced case is when n_components=12 and accuracy=0.90,
```

Comment:
In the generated docs, https://64501-843222-gh.circle-artifacts.com/0/doc/auto_examples/model_selection/plot_grid_search_refit_callable.html it generates:

```
The best_index_ is 2
The n_components selected is 10
The corresponding accuracy score is 0.88
```